docs/pages/admin-guides/access-controls/sso/okta.mdx

```diff
@@ -82,7 +82,70 @@ In this section we will create an application in the Okta dashboard to allow our
 Teleport cluster to access Okta as an IdP provider. We'll also locate the
 address that Okta uses to provides their IdP metadata to Teleport.
 
-(!docs/pages/includes/okta-create-saml-connector.mdx hosted="true"!)
+### Create Okta SAML 2.0 App
+
+From the main navigation menu, select **Applications** -> **Applications**, and click
+**Create App Integration**. Select SAML 2.0, then click **Next**.
+
+![Create APP](../../../../img/sso/okta/okta-saml-1.png)
+
+On the next screen (**General Settings**), provide a name and optional logo for
+your new app, then click **Next**. This will bring you to the **Configure SAML** section.
+
+### Configure the App
+
+Provide the following values to their respective fields:
+
+#### General
+
+- Single sign on URL: `https://<cluster-url>:<port>/v1/webapi/saml/acs/okta`
```

Comment:
Consider saying `cluster-hostname` or `cluster-endpoint` here.

The cluster "URL" would include https://..., and we don't want that twice.